meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8mmevk/bootloader.uuu.in

```diff
@@ -1,23 +1,20 @@
 uuu_version 1.2.39
 
-SDP: boot -f imx-boot-mfgtool
+SDP: boot -f SPL-mfgtool
 
-# These commands will be run when use SPL and will be skipped if no spl
-# SDPU will be deprecated. please use SDPV instead of SDPU
-SDPU: delay 1000
-SDPU: write -f imx-boot-mfgtool -offset 0x57c00
-SDPU: jump
-
-# These commands will be run when use SPL and will be skipped if no spl
-# if (SPL support SDPV)
 SDPV: delay 1000
-SDPV: write -f imx-boot-mfgtool -skipspl
+SDPV: write -f u-boot-mfgtool.itb
 SDPV: jump
 
 FB: ucmd setenv fastboot_dev mmc
 FB: ucmd setenv mmcdev ${emmc_dev}
 FB: ucmd mmc dev ${emmc_dev}
-FB: flash bootloader ../imx-boot-@@MACHINE@@
+FB: ucmd setenv set_blkcnt 'setexpr blkcnt 0x${filesize} + 0x1ff && setexpr blkcnt ${blkcnt} / 0x200'
+FB: ucmd setenv fastboot_buffer ${loadaddr}
+FB: download -f ../SPL-@@MACHINE@@
+FB: ucmd run set_blkcnt && mmc write ${loadaddr} 0x42 ${blkcnt}
+FB: download -f ../u-boot-@@MACHINE@@.itb
+FB: ucmd run set_blkcnt && mmc write ${loadaddr} 0x300 ${blkcnt}
 FB: ucmd if env exists emmc_ack; then ; else setenv emmc_ack 0; fi;
-FB: ucmd mmc partconf ${emmc_dev} ${emmc_ack} 1 0
+FB: ucmd mmc partconf ${emmc_dev} ${emmc_ack} 7 0
```

Comment:
What is `7` here?

Reply:
set user partition (with ack) as bootable